ecl2df/config_jobs/ECL2CSV

```diff
@@ -1,5 +1,16 @@
 EXECUTABLE ecl2csv
 
-ARGLIST <SUBCOMMAND> "--verbose" "--output" <OUTPUT> <ECLBASE>
+DEFAULT <XARG1> ""
+DEFAULT <XARG2> ""
+DEFAULT <XARG3> ""
+DEFAULT <XARG4> ""
+DEFAULT <XARG5> ""
+DEFAULT <XARG6> ""
+DEFAULT <XARG7> ""
+DEFAULT <XARG8> ""
+DEFAULT <XARG9> ""
+DEFAULT <XARG10> ""
+
+ARGLIST <SUBCOMMAND> "--verbose" "--output" <OUTPUT> <XARG1> <XARG2> <XARG3> <XARG4> <XARG5> <XARG6> <XARG7> <XARG8> <XARG9> <XARG10> "--" <ECLBASE>
```

Comment:
I don't mind 🤷 What we have done in libres forward models is to support `<OPTS>` defaulted to `""` and then you just pack all of the extra stuff into that one arguments...

Reply:
Hmm, maybe I forgot to try that..

Reply:
But does that work without a custom argparse?

Reply:
If I put say `"--time_index yearly"` in a forward model argument, ERT will complain:
`Error message: subst_list_add_from_string: something wrong with:<XARG1>=--time_index yearly  - spaces are not allowed in key or value part.`

Reply:
Pending review, I suggest merging this and injecting for komodo 2021.02.